### ISSUE TITLE
saved model

### ISSUE THREAD

**Issue by saria85** [Bartzi/see#62]
Thanks for sharing your code with us.
Could you also share your saved model?

**Comment by Bartzi** [comment]
I already shared the model, please have a look at [this website](https://bartzi.de/research/see), and also read the correspinding section of the [README](https://github.com/Bartzi/see#pretrained-models) :wink:

**Comment by harshalcse** [comment]
@Bartzi  Can we build model for CAPITAL ALPHANUMERIC Character string recognition ?

**Comment by Bartzi** [comment]
Sure :wink:
Just find the necesary training data and use create a new `char_map` only with the ascii codes for captial alphanumerics. Train and enjoy, yhis should be it...

**Comment by harshalcse** [comment]
@Bartzi so what is preriqusite of dataset preparation like image resolution and image size and image quality...

**Comment by Bartzi** [comment]
Hi,

good question. I never really cared about image resolution or quality, I mean the text should be readable, but that is about it. All samples that you put into the network will be resized to`200x64` pixels (see [here](https://github.com/Bartzi/see/blob/master/chainer/train_text_recognition.py#L59)), regardless of their original aspect ratio.

So, you will need samples that only contain the already cropped text line and a ground truth file together with the samples. You can find a sample ground truth [here](https://bartzi.de/research/see).

**Comment by harshalcse** [comment]
@Bartzi 
Is it required to create Dataset with bounding box file that is digitStruct.mat as it is there is SVHN dataset and if it is yes that how we can generate it ? 

Please help.

**Comment by Bartzi** [comment]
No, there is no need for bounding boxes! This is the point of the paper, you only need an image and the textual content as annotation. The README and the website I pointed you to should contai all information you need.

**Comment by harshalcse** [comment]
> No, there is no need for bounding boxes! This is the point of the paper, you only need an image and the textual content as annotation. The README and the website I pointed you to should contai all information you need.

@Bartzi I am asking for retraining and fine-tuning.

**Comment by Bartzi** [comment]
I'm sorry but I don't see your problem. If you want to retrain, or finetune, you just have to prepare everything as if you'd like to train the model from scratch, but you can just use an earlier snapshot as a starting point.

**Comment by harshalcse** [comment]
@Bartzi Sir the trained model provided is trained on which dataset please suggest and also please suggest using which .py file that model is trained.
